src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -134,6 +134,13 @@ def load_arguments(self, _):
         c.argument('disk_mbps_read_write', type=int, min_api='2018-06-01', help="The bandwidth allowed for this disk. Only settable for UltraSSD disks. MBps means millions of bytes per second with ISO notation of powers of 10")
         c.argument('upload_size_bytes', type=int, min_api='2019-03-01',
                    help='The size (in bytes) of the contents of the upload including the VHD footer. Min value: 20972032. Max value: 35183298347520')
+        c.argument('max_shares', type=int, help='The maximum number of VMs that can attach to the disk at the same time. Value greater than one indicates a disk that can be mounted on multiple VMs at the same time')
```

Comment:
is there constraint on this int value?

Reply:
Hard to put it into help. Too long.
#name: "P15", minsizeGiB: 128, maxsizeGiB: 256, miniops: 1100, maxiops: 1100, minbandwidthMBps: 125, maxbandwidthMBps: 125, maxValueOfMaxShares: 2
#name: "P20", minsizeGiB: 256, maxsizeGiB: 512, miniops: 2300, maxiops: 2300, minbandwidthMBps: 150, maxbandwidthMBps: 150, maxValueOfMaxShares: 2
#name: "P30", minsizeGiB: 512, maxsizeGiB: 1024, miniops: 5000, maxiops: 5000, minbandwidthMBps: 200, maxbandwidthMBps: 200, maxValueOfMaxShares: 5
#name: "P40", minsizeGiB: 1024, maxsizeGiB: 2048, miniops: 7500, maxiops: 7500, minbandwidthMBps: 250, maxbandwidthMBps: 250, maxValueOfMaxShares: 5
#name: "P50", minsizeGiB: 2048, maxsizeGiB: 4096, miniops: 7500, maxiops: 7500, minbandwidthMBps: 250, maxbandwidthMBps: 250, maxValueOfMaxShares: 5
#name: "P60", minsizeGiB: 4096, maxsizeGiB: 8192, miniops: 16000, maxiops: 16000, minbandwidthMBps: 500, maxbandwidthMBps: 500, maxValueOfMaxShares: 10
#name: "P70", minsizeGiB: 8192, maxsizeGiB: 16384, miniops: 18000, maxiops: 18000, minbandwidthMBps: 750, maxbandwidthMBps: 750, maxValueOfMaxShares: 10
#name: "P80", minsizeGiB: 16384, maxsizeGiB: 32767, miniops: 20000, maxiops: 20000, minbandwidthMBps: 900, maxbandwidthMBps: 900, maxValueOfMaxShares: 10